src/raw/lined/clock-rewind.svg

```diff
@@ -1,4 +1,4 @@
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
-  stroke-linejoin="round" width="24" height="24" fill="none">
```

Comment:
Was this `fill="none"` intentional?  For all lined icons we do not want to allow any fills

Reply:
Yes it was intentional, the attribute was set twice and `sharp` caught it.

I thought about suppressing issues when generating PNGs if they failed to generate but this one seemed like it could be helpful and I figured the extra validation might be nice if we have invalid SVGs in the future. I also figured we could always add error suppression later if the validation becomes a nuisance.

Reply:
Doh - I see it now, thanks!